docs/reference/cat/indices.asciidoc

```diff
@@ -13,8 +13,9 @@ green  open   twitter2   2   0       2030            0      5.8mb          5.8mb
 --------------------------------------------------
 
 We can tell quickly how many shards make up an index, the number of
-docs, deleted docs, primary store size, and total store size (all
-shards including replicas).
+docs at Lucene level (eg.: including nested documents), deleted docs, 
```

Comment:
Can we change this to say "at the Lucene level (including hidden docs from e.g., nested docs)"?